toolkits/global/packages/global-buttons/scss/10-settings/buttons.scss

```diff
@@ -0,0 +1,77 @@
+/**
+ *  Button settings
+ *
+ */
+
+$button-base--text-align: left;
+$button-base--padding: 8px;
+$button-base--line-height: 1.6;
+$button-base--border-radius: 2px;
+$button-shared--border-width: 1px;
+$button-shared--border-style: solid;
+$button-shared--transition: 0.25s ease, color 0.25s ease, border-color 0.25s ease;
+
+/**
+ * Button Themes
+ * Variables to set themes styles
+ *
+ */
+
+$button--default: (
+	'color': color('action-base'),
+	'backgroundColor': greyscale(95),
+	'backgroundImage': linear-gradient(to bottom, $page-base, greyscale(95)),
+	'borderColor': greyscale(80),
+	'visitedColor': color('action-base'),
+	'hoverColor': white,
+	'hoverBorderColor': greyscale(40),
+	'hoverBackgroundColor': greyscale(40),
+	'hoverBackgroundImage': none,
+	'hoverTextDecoration': none,
+	'focusBorderColor': greyscale(40),
+	'focusTextDecoration': none
+);
+
+$button--ghost: (
```

Comment:
I'm wondering what the ghost modifier is for?

Reply:
IE is it a name/distinction that all brands will potentially use?

Reply:
It was the name used for buttons that have a transparent background and with visible border on button. I added a line in the readme to describe but can add to that if that was too vague

Reply:
Actually some of these themes including ghost probably not be in global and I also need to neutralise the colors so they are not springer specific.

Reply:
ah ok. yes I wasn't sure if that appearance was going to be global. Maybe we could just specify a primary and secondary style that can be overridden at brand level? Ghost seems a little prescriptive maybe?

Reply:
Yep totally agree. I'll have another PR for the brand level stuff after this.

Reply:
Good stuff 👍